smap/sources/safarScraper.py

```diff
@@ -6,6 +6,7 @@
 # used for logging
 import logging
 import sys, traceback
+import simplejson as json
```

Comment:
Do you really need to use simplejson? I thought that python has builtin json support now...